src/Storage/TSStorageManager.m

```diff
@@ -197,8 +197,10 @@ - (void)setupDatabase
 {
     // Register extensions which are essential for rendering threads synchronously
     [TSDatabaseView registerThreadDatabaseView];
-    [TSDatabaseView registerBuddyConversationDatabaseView];
+    [TSDatabaseView registerThreadInteractionsDatabaseView];
```

Comment:
Sorry, which migration?  That's a really key scenario to test with this work (I've already been testing it a ton), but this particular database view isn't changing - I've just renamed it in the code, but not changed its extension name.